packages/jest-matcher-utils/src/__tests__/deepCyclicCopyReplaceable.test.ts

```diff
@@ -20,20 +20,6 @@ test('returns the same value for primitive or function values', () => {
   expect(deepCyclicCopyReplaceable(fn)).toBe(fn);
 });
 
-test('does not execute getters/setters, but copies them', () => {
```

Comment:
So it's not just in the comments - we invoke the getters on purpose now, so this test doesn't make sense anymore